### PR TITLE
[IMP] codeowners: add framework js codeowners

### PR DESCRIPTION
**Pull request by SimonGenin**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -151,3 +151,11 @@
...
 /odoo/http.py @odoo/rd-security @odoo/rd-website
 /odoo/sql_db.py @odoo/rd-security
+
+/addons/bus @odoo/rd-framework-js
+/addons/web/ @odoo/rd-framework-js
+/addons/web_kanban_gauge/ @odoo/rd-framework-js
+/odoo/addons/base/models/assetsbundle.py @odoo/rd-framework-js
+/odoo/cli/ts_config.py @odoo/rd-framework-js
+/odoo/tools/js_transpiler.py @odoo/rd-framework-js
+/odoo/tools/sourcemap_generator.py @odoo/rd-framework-js
```
Comment (seb-odoo) [inline .github/CODEOWNERS:161]: Hello, order is this file is very important (last matching rule has priority), so we enforce rather strict guidelines (see comments at the top of the file).

In this case, they should be sorted alphabetically among other modules after the `# Specific rules` line.

**Review by seb-odoo — CHANGES_REQUESTED** [review]
Reminder: the group @odoo/rd-framework-js must exist and must have the access rights on the repo (this needs to be done by someone have full admin on our github).

**Comment by seb-odoo** [comment]
Also a note: targeting master will make you miss all bugfixes, and also miss all the foward-port of all bugfixes (because they are created as draft to avoid noise).

If you want to catch them too, the PR should target v12. But this is a choice to make for your team, no enforced guideline here.

**Comment by SimonGenin** [comment]
@seb-odoo Do you know who has full access to the repo to give our team the rights ?

**Comment by seb-odoo** [comment]
saas/internal teams could help you with that

**Comment by SimonGenin** [comment]
moved to v12

**Comment by KangOl** [comment]
> Reminder: the group @odoo/rd-framework-js must exist and must have the access rights on the repo (this needs to be done by someone have full admin on our github).

Which rights are you talking about?

**Comment by seb-odoo** [comment]
@KangOl The group itself must have "write" rights on the repo. I'm not sure how it's done on github or the exact terms used to do it, but you can check other groups that are in codeowners to see how its done. Olivier should be able to help if needed as he set up the first ones.

Note I think every member of the group that indent to be codeowners/reviewer needs the rights too. I'm not sure if newer employees get them by default nowadays.

**Pull request closed without merging**